addons/hovercards/addon.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "Hovercards",
+  "description": "Adds informational hovercards for links to projects, users, studios, forum topics, and forum posts. Uses the Scratch API and ScratchDB.",
+  "credits": [
+    {
+      "name": "Hans5958"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "script.js",
+      "matches": "*"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "base.css",
+      "matches": "*"
+    },
+    {
+      "url": "scratch2.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "scratch2"
+        }
+      }
+    },
+    {
+      "url": "scratch3.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "scratch3"
+        }
+      }
+    },
+    {
+      "url": "extended.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "extended"
+        }
+      }
+    },
+    {
+      "url": "extended-statistics.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "extendedStatistics"
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "id": "theme",
+      "name": "Theme",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "scratch3",
+          "name": "Scratch 3"
+        },
+        {
+          "id": "scratch2",
+          "name": "Scratch 2"
+        },
+        {
+          "id": "extended",
+          "name": "Extended"
+        },
+        {
+          "id": "extendedStatistics",
+          "name": "Extended (statistics)"
```

Comment:
Is this really a theme?

Reply:
Doesn't look like it is. "Theme" should only affect the style. So this should be split to 1) Style setting (www vs r2) and 2) Display mode (simple, extended, statistics)

Reply:
There should be no setting for theme/style - all tooltips should be scratch-www styled on scratch-www pages and scratchr2-styled on scratchr2 pages. I think profile pictures of project authors could be always visible.

Reply:
I agree with this. (It would also be scratch-www style on 2.0->3.0-ified pages.)